src/Components/Patient/PatientHome.tsx

```diff
@@ -400,11 +400,13 @@ export const PatientHome = (props: any) => {
       <div id="revamp">
         <PageTitle
           title={"Patient Details"}
-          backUrl="/patients"
           crumbsReplacements={{
             [facilityId]: { name: patientData?.facility_object?.name },
             [id]: { name: patientData?.name },
           }}
+          backUrl={
+            facilityId ? `/facility/${facilityId}/patients` : `/patients`
+          }
```

Comment:
@Pragati1610 suppose I'm on: [`/facility/${facility_id}/patient/${patient-id}/consultation/${consultation-id}`](https://deploy-preview-3637--care-net.netlify.app/facility/7ab27974-32c8-4e36-bdf9-dbf102524718/patient/cc52390c-ac8a-4af5-9806-f3df7ee1795a/consultation/7037609e-3c25-45ed-bfd8-e01c346cbd45)
and click on Patient Details and go back, it takes me back to Patients list page instead of the patient consultation page.
![image](https://user-images.githubusercontent.com/25143503/193835386-c8673baa-702a-4163-91c3-5584a516619f.png)